cves/kernel/CVE-2014-2851.yml

```diff
@@ -55,7 +55,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  There was improper counters in a for loop, possibly leading to integer overflow. 
```

Comment:
counter? Can you provide more background on this vulnerability?

Reply:
I'll try.